Packs/XForceExchange/Integrations/XFE_v2/XFE_v2.py

```diff
@@ -472,10 +476,20 @@ def main():
     params = demisto.params()
     credentials = params.get('credentials')
 
+    reliability = demisto.params().get('integrationReliability')
+    reliability = reliability if reliability else DBotScoreReliability.B

```

Comment:
Is the reliability of XForce B or C?
```suggestion
    reliability = reliability if reliability else DBotScoreReliability.C

```